scripts/install/configuration/apply_csm_configuration.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+usage()
+{
+   # Display Help
+   echo "Runs CFS to setup passwordless ssh"
+   echo "All parameters are optional and the values will be determined automatically if not set."
+   echo
+   echo "Usage: deploy_ssh_keys.sh [ --csm-release version ] [ --git-commit id ]"
```

Comment:
I think the actual term is git hash, not id, but not a big deal. Maybe at least use hash in the argument description below.

Reply:
Fixed